docs/kraken-x3-z3-guide.md

```diff
@@ -172,6 +172,8 @@ _New in git._<br>
 
 Adds support for NZXT Kraken 2023 Standard, Elite
 
+*For firmware version 2 gif screen mode is no longer supported*
```

Comment:
Maybe link the issue #631 here, and do say that this only applies to 2023 models.

Reply:
How do you link issues? just hashtag in code?

Reply:
Yes, something like `*On 2023 models (standard and ELite), GIF screen mode is no longer supported for firmware versions 2.X (see #631).*

Reply:
Actually wait, I think the syntax is different.

Reply:
Well, I copied this already, and it's linked
I can change it of course

Reply:
This works:

```
*On 2023 models (standard and Elite), GIF screen mode is no longer supported for firmware versions 2.X (see [#631][`issue-631`]).*
```

With this at the end of the file (it already has a reference to liquidtux there, so below that):

```
[`issue-631`]: https://github.com/liquidctl/liquidctl/issues/631
```

Sorry, I forgot that it doesn't automatically do that for md files.

> Well, I copied this already, and it's linked

In the PR view it is, but not when you access the file [normally](https://github.com/liquidctl/liquidctl/blob/c77f20306b071d97c2b92069f7c6850bca953770/docs/kraken-x3-z3-guide.md#the-lcd-screen-only-z-and-2023-models).